monitor_memory.py

```
@@ -0,0 +1,363 @@
+            print(f"❌ Failed to fetch metrics: {e}")
+            return None
+
+    def check_health(self) -> Optional[Dict[str, Any]]:
+        """Check if API is healthy."""
+        try:
+            response = requests.get(
+                f"{self.api_url}/health",
+                timeout=5
+            )
+            return response.json() if response.status_code == 200 else None
+        except:
```

Ruff (0.14.2)

Do not use bare `except`

(E722)